project/Dependencies.scala

```diff
@@ -56,13 +56,22 @@ object Dependencies {
     )
   }
 
+
+  val MongoDb = Seq(
+    libraryDependencies ++= Seq(
+      "org.mongodb.scala" %% "mongo-scala-driver" % "1.2.1", // ApacheV2
+      "com.typesafe.play" %% "play-json" % "2.6.0-M1"
```

Comment:
Would be better if it didn't depend on a milestone, if it is to depend on a json library at all.